packages/disploy/src/structs/UserContextMenuInteraction.ts

```diff
@@ -9,7 +9,7 @@ export class UserContextMenuInteraction extends ContextMenuInteraction {
 	 */
 	public readonly targetUser: User;
 
-	public constructor(app: App, raw: APIUserApplicationCommandInteraction) {
+	public constructor(app: App, public override raw: APIUserApplicationCommandInteraction) {
```

Comment:
While this would probably be a good idea to let the user access the raw objects; it is out of scope for this PR.